### PR TITLE
user delete perms

### PR DESCRIPTION
**Pull request by jjidt**
-  also adds a validation enum for org role id updates on the user

### CODE REVIEW

File: api/src/constants/acceptableRoleIdUpdateValues.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export default Object.freeze([2, 3])
```
Comment (dheniges) [inline api/src/constants/acceptableRoleIdUpdateValues.js]: should use the organization role id constants

**Review by dheniges — DISMISSED** [review]
Looks good, just one constant update.

Tested all the deletes and everything worked as expected

File: api/src/constants/acceptableRoleIdUpdateValues.js (hunk 1 above)
Comment (jjidt) [reply api/src/constants/acceptableRoleIdUpdateValues.js]: ah good call

**Review by dheniges — APPROVED** [review]
Re-approved

**Pull request merged**